translate.js

```diff
@@ -697,8 +709,11 @@ export default (theme) => (str) => {
             case 'right':
               out['background-position'] = `${i[1]} ${i[2]}`;
               break;
+            case 'opacity':
+              out['--ow-bg-opacity'] = i[2] / 100;
```

Comment:
This method of setting opacity was simplest though it does give more options than Tailwind provides. Not sure if that's an issue or not. I figure if someone wants to set 47% opacity then they're free to do so with this shorthand.

Reply:
This is one of those weird tradeoffs. By the nature of this runtime approach, we can do things like this.. property values can be very dynamic. But is that why people use tailwind.. or do they like the theme constraints? Does working within those constraints help consistency for example?

If you want to work within the theme restricted paradigm then you could change this to:

```js
out['--ow-bg-opacity'] = theme.opacity[i[2]] / 100;
```

Does that make sense?

Reply:
Sure, will do.